packages/main/src/pages/static/contact-us/components/ContactFormContainer.jsx

```diff
@@ -31,14 +31,14 @@ function ContactFormContainer() {
     success: ""
   });
 
-  useEffect(() => {
-    let userInfo = getUserInfo();
-    setUserAuth(userInfo.user.email ? userInfo.user : {});
-    setValues(values => ({
-      ...values,
-      email: userAuth.email ? userAuth.email : values.email
-    }));
-  }, []);

```

Comment:
Please explain why you removed this code and what code you added to substitute this functionality. I'm trying to see the thought process behind this and why this code would not work